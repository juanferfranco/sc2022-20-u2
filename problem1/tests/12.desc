Intenta adicionar un evento que ya existe