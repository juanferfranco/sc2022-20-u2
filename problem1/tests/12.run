./main < tests/12.in
