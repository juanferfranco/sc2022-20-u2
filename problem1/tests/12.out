ev1
ev2
ev3
ev4
ev5
ev6
ev7
ev8
ev1
ev2
ev3
ev4
ev5
ev6
ev7
ev8
